vsc_pdf_template_transformer/lib/models/tpl_box_fit.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/widgets.dart' as ws;
+import 'package:vsc_pdf_template_transformer/utils/evaluator.dart';
+
+part 'tpl_box_fit.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplBoxFit {
```

Comment:
It doesn't look like we need a separate class for this. This requires the template JSON to have:
```json
  "fit": { 
    "fit": "'cover'"
}
```
Rather than just saying:
```json
  "fit": "'cover'"
```

TplLimitedBox should just call  `evaluateBoxFit()` directly.